### PR TITLE
fix(modify-position): improve modify positions styles

### PR DESCRIPTION
**Pull request by FiboApe**

### CODE REVIEW

File: packages/ui-library/src/components/divider/index.tsx [hunk 1]
```diff
@@ -1,4 +1,36 @@
...
+  ${({
+    theme: {
+      palette: { mode },
+    },
+  }) => `
+    border-color: ${colors[mode].border.border1};
+  `}
+`;
+
+const DividerBorder2 = styled(Divider)`
```
Comment (th0rOdinson) [inline packages/ui-library/src/components/divider/index.tsx:26]: Like it very much

File: packages/ui-library/src/components/modal/index.tsx [hunk 2]
```diff
@@ -182,14 +167,19 @@ const Modal: React.FC<ModalProps> = ({
...
                 key={index}
+                sx={{ width: '100%' }}
               >
                 {action.label}
               </Button>
             )
           )}
-        </StyledDialogActions>
+          {showCloseButton && (
+            <Button onClick={onClose} variant="outlined" size="large" sx={{ width: '100%' }}>
```
Comment (th0rOdinson) [inline packages/ui-library/src/components/modal/index.tsx:178]: Doesn't native fullWidth work in thi case? Same for L171

File: packages/ui-library/src/theme/typography.ts [hunk 3]
```diff
@@ -104,7 +104,7 @@ export const buildTypographyVariant = (mode: PaletteMode): TypographyOptions =>
     fontSize: '1rem', // 16/16
     fontStyle: 'normal',
     fontWeight: 400,
-    lineHeight: 'normal', // 32/16
+    lineHeight: 2, // 32/16
```
Comment (th0rOdinson) [inline packages/ui-library/src/theme/typography.ts:107]: I believe this is true, but only when body is bold

File: packages/ui-library/src/components/modal/index.tsx (hunk 2 above)
Comment (FiboApe) [reply packages/ui-library/src/components/modal/index.tsx:178]: It adds a few more styles that we dont want

File: packages/ui-library/src/theme/typography.ts (hunk 3 above)
Comment (FiboApe) [reply packages/ui-library/src/theme/typography.ts:107]: Oh let me check that

**Review by th0rOdinson — APPROVED** [review]

**Pull request merged**